src/cart/crt_context.c

```diff
@@ -228,6 +229,34 @@ crt_context_create(crt_context_t *crt_ctx)
 
 	D_RWLOCK_UNLOCK(&crt_gdata.cg_rwlock);
 
+	/** initialize sensors */
+	if (crt_gdata.cg_use_sensors) {
+		int	ret;
+
+		ret = d_tm_add_metric(&ctx->cc_timedout, D_TM_COUNTER,
+				      "Total number of timed out RPC requests",
+				      "", "net/%u/req_timeout", ctx->cc_idx);
+		if (ret)
+			D_WARN("Failed to create timed out req counter: "DF_RC
+			       "\n", DP_RC(ret));
+
+		ret = d_tm_add_metric(&ctx->cc_timedout_uri, D_TM_COUNTER,
+				      "Total number of timed out URI lookup "
+				      "requests", "",
+				      "net/%u/uri_lookup_timeout", ctx->cc_idx);
+		if (ret)
+			D_WARN("Failed to create timed out uri req counter: "DF_RC
+			       "\n", DP_RC(ret));
+
+		ret = d_tm_add_metric(&ctx->cc_timedout_uri, D_TM_COUNTER,
```

Comment:
is this supposed to use same metric ctx->cc_timedout_uri here? it is already used on line 124.
feels it should be cc_failed_addr instead

Reply:
this should say line 243 and here

Reply:
fixed.